python/nano/src/bigdl/nano/utils/common/checker.py

```diff
@@ -62,3 +67,14 @@ def _avx512_checker():
     '''
     cpuinfo = CPUInfo()
     return cpuinfo.has_avx512
+
+
+def _avx2_checker():
+    # copied from https://github.com/intel/intel-extension-for-pytorch
+    try:
+        if compare_version("intel_extension_for_pytorch", operator.ge, "2.0.0"):
+            import intel_extension_for_pytorch._isa_help as isa
+            return isa._check_isa_avx2()
+    except SystemExit:
```

Comment:
we may meet `ImportError` here for non-ipex users right? This may cause some minor issues for a non-AVX2 && non-ipex user

Reply:
Thanks for reminder! will add ImportError here : )